content/python/concepts/time-module/sleep.md

```diff
@@ -0,0 +1,46 @@
+---
+
+Title: '.sleep()'
+
+Description: 'The function suspends execution of current thread for a given time.'
+
+Subject: 'Computer Science'
+
+Tags: 
```

Comment:
Tags need to be from the list [here ](https://github.com/Codecademy/docs/blob/main/documentation/tags.md). "Python" & "Time Module" aren't. Also should be "Functions", not "Function".

Also, just indent the list one space.